docs/millennium-os/manual/chapters/40_gcodes.md

```diff
@@ -81,7 +81,7 @@ M5010 W0 R5
 
 Reset the stored details for a given WCS, or the current WCS if not specified.
 
-Different fields are used for different types of probing operations, and we want to reset these values before running a probing cycle - so if previous values existed but the probing cycle failed we would not end up using the previous valid values.
+Different fields are used for different types of probing operations, and we want to reset these values before running a probing cycle, so if previous values existed, but the probing cycle failed we would not end up using the previously valid values.
```

Comment:
I'm not sure about this change to `previously` - the values are still valid, they're just from an 'old' probe cycle. `previously valid` sounds like the values were valid but are now not.

Maybe there's another way to write this that is clearer, without using previous or previously?

Reply:
The whole thing could use a bit of a rewrite for clarity. See if the below is a correct understanding of what's meant to be there:

> Not all fields are needed for every probing operation type, and we want to reset the old values before running a new probing cycle. If previous values exist and the probing cycle fails, we will not use the existing values.

Reply:
```suggestion
Not all fields are needed for every probing operation type, and we want to avoid using previously-stored values if the most recent probing operation has failed. To do that, we use this macro to clear the relevant values before starting each probing cycle.
```

Does this work? I think you're right about needing a rewrite but I'm struggling with the "existing values" thing.